src/App.jsx

```diff
@@ -13,23 +13,57 @@ const useSmiPersistantState = () => {
   return [todoList, setTodoList];
 }
 
-function App() {
-  
-  const [todoList, setTodoList] = useSmiPersistantState();
 
-  const addTodo = (newTodo) => {
+
+const App = () => {
+
+  const [todoList, setTodoList] = useSemiPersistentState();
```

Comment:
This was supposed to go back to using the `useState` hook, since you'll delete the function.